search/includes/classes/commands/class-healthcommand.php

```diff
@@ -262,20 +262,26 @@ private function render_results( array $results ) {
 	 * [--silent]
 	 * : Optional silences all non-error output except for the final results
 	 *
+	 * [--force-parallel-execution]
```

Comment:
I'm still not convinced we want to do locking on the CLI commands (even if it's bypass-able), but we can go with this and see if it ends up being an issue in practice.

Reply:
Along those lines, can we detect the "locked" error (with code `content_validation_already_ongoing`) in the error handler (line 287) and print out a message indicating it can be bypassed with this option?

Reply:
Good idea!